controller/app/helpers/admin_helper.rb

```diff
@@ -99,8 +99,14 @@ def get_user_info(user)
     user["ssh_keys"].each { |k| user_ssh_keys << ["#{user['_id'].to_s}-#{k['name']}", Digest::MD5.hexdigest(k["content"]), nil] if k["content"] } if user["ssh_keys"].present?
     user_caps = user["capabilities"]
     if user["plan_id"]
-      plan_caps = OpenShift::BillingService.instance.get_plans[user["plan_id"].to_sym][:capabilities]
-      user_caps = plan_caps.deep_dup.merge(user_caps)
+      plans = OpenShift::BillingService.instance.get_plans[user["plan_id"].to_sym]
```

Comment:
I believe this will still cause the error to be reported when `get_plans` returns `nil`. You cannot call `[]` on `nil`. Instead, call `[user["plan_id"].to_sym]` after you ensure that `get_plans` did not return `nil`.